### DISCUSSION TITLE
Paused job resumes by itself

### DISCUSSION THREAD

**Discussion by ClassyCircuit** [quartznet/quartznet#1512] [Q&A]
I paused all the jobs that are running on my server but then it decided to resume by itself, randomly for no apparent reason.

What is causing this behaviour and how to stop it?

If I pause all the triggers, then I expect them to stay paused until I resume them myself.

**Comment by lahma** [comment]
You probably need to enable logging and inspect the logs. There's no built-in functionality in Quartz to resume jobs without interaction.

**Comment by ClassyCircuit** [comment]
Could it be related to IScheduler lifetime? 
If I create the SchedulerFactory instance manually every time in constructor and then get the schedule, could it be related to this behaviour?

What is the correct way of using SchedulerFactory and IScheduler? 
Should I register "StdSchedulerFactory" as a scoped service during startup and then retrieve my scheduler whenever needed? 
Or is there another way?

**Reply by lahma** [reply]
Scheduler factory should be created only once, you can call GetScheduler() as many times as  needed when you need a reference. Hosting package contains all APIs to handle the details. 

https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/hosted-services-integration.html

The example apps in repo show more usage examples.

**Comment by ClassyCircuit** [comment]
Thanks
